web/modules/custom/mymodule/mymodule.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'My Module'
+description: 'This is the first custom module.'
+type: module
+core_version_requirement: ^10
+package: 'Custom package'
+dependencies:
+  - admin_toolbar:admin_toolbar
+  - token
```

Comment:
This red circle means new line is missing at the end of file.
Please fix this for all files.

Reply:
Resolved.